### DISCUSSION TITLE
Will this work reliably with Valkey >8.0 going forward?

### DISCUSSION THREAD

**Discussion by txtsd** [rhubarbgroup/.github#57] [Support (ROC)] [answered]
Valkey will not always be backward compatible with Redis.
Will Redis Object Cache and Object Cache Pro work seamlessly with Valkey >8.0 going forward?

https://valkey.io/blog/valkey-8-ga/

**Comment by tillkruss** [comment, marked as answer]
Yes, we'll primarily support Valkey.

**Reply by txtsd** [reply]
Awesome! Thanks!
